specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -2123,6 +2124,24 @@
           "description": "Percentage completed.",
           "readOnly": true
         },
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The Azure Resource ID."
+              }
+            }
+          },
+          "description": "A list of Azure Resource IDs."
```

Comment:
This isn't necessarily a list of resource ids (e.g., for LAC calls). Might just want to say that this is a list of results for async operations that return a list of results.